bins/revme/src/cmd/evmrunner.rs

```diff
@@ -25,6 +25,14 @@ pub enum Errors {
     EVMError,
     #[error(transparent)]
     Io(IoError),
+    #[error(transparent)]
+    EofError(EofDecodeError),
+}
+
+impl From<EofDecodeError> for Errors {
```

Comment:
i believe you can do EofError(#[from] EofDecodeError) in the enum

Reply:
That is better